actions/release/notes/entrypoint

```diff
@@ -1,13 +1,90 @@
 #!/bin/bash
 
+set -e
+set -u
+set -o pipefail
+
 function main() {
+  local token repo
+  while [ "${#}" != 0 ]; do
+    case "${1}" in
+      --token)
+        token="${2}"
+        shift 2
+        ;;
+
+      --repo)
+        repo="${2}"
+        shift 2
+        ;;
+
+      "")
+        shift
+        ;;
+
+      *)
+        echo "unknown argument \"${1}\""
+        exit 1
+    esac
+  done
+
+  if [[ -z "${token}" ]]; then
+    echo "--token is a required flag"
+    exit 1
+  fi
+
+  if [[ -z "${repo}" ]]; then
+    echo "--repo is a required flag"
+    exit 1
+  fi
+
   local content
   content="$(
     jam summarize \
       --buildpack "${GITHUB_WORKSPACE}/build/buildpackage.cnb" \
       --format markdown
   )"
 
+  local version
+  version="$(
+    curl "https://api.github.com/repos/${repo}/releases/latest" \
+      --header "Authorization: token ${token}" \
+      --location \
+      --silent \
+    | jq -r -S .tag_name
+  )"
+
+  IFS=$'\n' read -r -d '' -a commits < <(
+    curl "https://api.github.com/repos/${repo}/compare/${version}...main" \
+      --header "Authorization: token ${token}" \
+      --location \
+      --silent \
+    | jq -r -S .commits[].sha \
+    && printf '\0' # NULL-terminate the input
+  )
+
+  local changes
+  changes="$(
+    for commit in "${commits[@]}"; do
+      curl "https://api.github.com/repos/${repo}/commits/${commit}/pulls" \
+        --header "Accept: application/vnd.github.groot-preview+json" \
+        --header "Authorization: token ${token}" \
+        --location \
+        --silent \
+      | jq -r -S -c '.[] | @text "* #\(.number): \(.title) *@\(.user.login)*"'
+    done \
+    | sort -n
+  )"
+
+  if [[ -n "${changes}" ]]; then
+    content="$(
+      echo "${content}"
+      echo
+      echo "### Changes"
+      echo "${changes}"
+    )"
+  fi
```

Comment:
Would it be possible for us to have some kind of check here to exclude any github-config bump commits?

This is fabulous, but the only part I am concerned about is the `Update github-config @paketo-bot` commits that get added. 

The packit bumps are useful, but to users looking at the release notes, the fact that the github-config got updated doesn't help much with anything. Also, an outside user might not know what that is, and it can make the changelog more difficult to understand.

Reply:
Done.